src/info/author.rs

```diff
@@ -70,19 +70,43 @@ impl AuthorsInfo {
         let authors = git_metrics.authors_to_display.clone();
         Self { authors }
     }
+
+    fn top_contribution(&self) -> usize {
+        if let Some(top_contributor) = self.authors.get(0) {
+            return top_contributor.contribution;
+        }
+        0
+    }
+}
+
+fn digit_difference(num1: usize, num2: usize) -> usize {
+    let count_digits = |num: usize| {
+        if num == 0 {
+            1
+        } else {
+            (num.checked_ilog10().unwrap() + 1) as usize
+        }
```

Comment:
```suggestion
        (num.checked_ilog10().unwrap_or(0) + 1) as usize
```
`checked_ilog10` returns `None` if `num` is `0`.

Reply:
thanks!